src/utils/date.test.ts

```diff
@@ -0,0 +1,34 @@
+import { getDateByTimeString, getDifferenceBetweenDate } from "./date";
+
+test("Testing difference between dates - no difference", () => {
+  expect(getDifferenceBetweenDate(new Date("2019-05-16 00:00:00"))).toEqual(0);
+});
+
```

Comment:
can we have unified lining? either have empty line after tests or not. I would prefer having an empty line between every test